docker-compose.yml

```diff
@@ -25,6 +25,27 @@ services:
     depends_on:
       - mssqlserver
 
+  order: 
+    build:
+      context: .\OrderApi
+      dockerfile: Dockerfile
+    image: eventbrite/orderapi
+    environment: 
+      IdentityUrl: http://10.10.10.30:5070
+      SQLSERVER_Host: catalogdb
+      SA_PASSWORD: ProductApi(!)
```

Comment:
Could you actually just remove your changes to docker-compose so it doesn't overwrite what we currently have?

Reply:
okay I will do that

Reply:
I didn't do migration thinking I would it after the PR but I will delete the ones that are already there 
```